.github/workflows/tests.yml

```
@@ -8,14 +12,69 @@ on:
       - 'release/**'
 
 jobs:
-  build-dev:
-    runs-on: ubuntu-18.04
+  build:
+    name: Build and push image
+    runs-on: ubuntu-20.04
+
     steps:
-      - uses: actions/checkout@v2
-      - name: Prints podman --version
-        run: podman --version
-      - name: Build dev
-        run: make dev.build
+    - uses: actions/checkout@v2
+      with:
+        # Nix Flakes doesn't work on shallow clones
+        fetch-depth: 0
+#    - name: Build Image
+#      id: build-image
+#      uses: redhat-actions/buildah-build@v2
+#      with:
+#        image: ${{ github.repository }}/python
+#        tags: dev-latest
+#        dockerfiles: |
+#          ./Containerfile.dev
+
+    - uses: cachix/install-nix-action@v13
+      with:
+        install_url: https://nixos-nix-install-tests.cachix.org/serve/i6laym9jw3wg9mw6ncyrk6gjx4l34vvx/install
+        install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
+        extra_nix_config: |
+          experimental-features = nix-command flakes ca-references ca-derivations
+          sandbox = false
+    - run: nix flake --version
+#    - run: nix flake metadata nixpkgs
+
+    # Remove podman via apt-get
+    - run: sudo apt-get purge -y podman containers-image containers-common
+    - run: sudo apt-get autoremove --purge
+    - run: sudo apt-get -y clean
+    - run: podman --version || echo podman not installed
```

Pode fazer um pipe de comandos no CI
```suggestion
    - run: |
        sudo apt-get purge -y podman containers-image containers-common
        sudo apt-get autoremove --purge
        sudo apt-get -y clean
        podman --version || echo podman not installed
```